### PR TITLE
Updates branch

### PR DESCRIPTION
**Pull request by direakanbi**
# My portfolio
This project was created for the creation of my portfolio.

# SITE CONTENTS
1. A home section.
2. A project showcase section.
3. An About Me section.

# LANGUAGES USED
1. HTML
2. CSS
3. Linters(Lighthouse, Webhint, Stylelint
# PREREQUISITES
  1.Prior knowledge of HTML and CSS.

  # Live Demo Link.
  http://127.0.0.1:5500/portfolio1/index.html#

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
 This project was created for the creation of my portfolio.
 
+# SITE CONTENTS
+1. A home section.
+2. A project showcase section.
+3. An About Me section.
+
 # LANGUAGES USED
 1. HTML
 2. CSS
```
Comment (Mihndim2020) [inline README.md]: - [ ] Please your Readme file is not descriptive enough, kindly follow this [template](https://github.com/microverseinc/readme-template), personalize it and fill out the available information.

File: index.html [hunk 2]
```diff
@@ -9,36 +9,214 @@
...
+          <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+          <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+          <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+          <li><a href="#" aria-label="github"><i class="fab fa-github"></i></a></li> 
         </ul>
       </div>
     </section>
+      
+    <section>
+      <ul class="cards-container">
```
Comment (Mihndim2020) [inline index.html:46]: - [ ] The container of this section is a `<section>` tag. This is part of the project requirements and it makes your html more semantic, which good for assistive technologies like screen readers, it also enhances search engine optimization. Check out this [article](https://www.w3schools.com/html/html5_semantic_elements.asp).
- [ ] The main container of the projects should have a `display:grid`. Kindly fix that.

File: index.html [hunk 3]
```diff
@@ -9,36 +9,214 @@
...
+        <li class="work one">
```
Comment (Mihndim2020) [inline index.html:47]: - [ ] For each project ("work") content of the project is aligned with flexbox. Please kindly implement that.

File: index.html [hunk 4]
```diff
@@ -9,36 +9,214 @@
...
+          </p>
+        </section>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+        <button class="see">See Project</button>
```
Comment (Mihndim2020) [inline index.html:71]: - [ ] The button should follow the button styling: Active, Hover and Pressed (see design below) and kindly fix that.
buttonsat
 - [ ] The section should have the margins (top, bottom, left, right) defined in the design. Please checkout the difference below and kindly fix yours. 
 
 **Provided design**
dire1

**Your implementation**
dire2

File: index.html [hunk 5]
```diff
@@ -9,36 +9,214 @@
...
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+         <button class="see">See Project</button>
+        
+        </li>
+
+      <li class="about">
```
Comment (Mihndim2020) [inline index.html]: - [ ] The container of this section is a `<section>` tag, as it is a requirement and for semantic purposes as explained above. 
- [ ] The content should be aligned with flexbox. Please kindly fix that.

File: index.html [hunk 6]
```diff
@@ -9,36 +9,214 @@
...
+        <h2>About <br>Myself</h2>
+        <p class="about1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <div class="contacts"> 
+          <a href="#" class="lets-connect">LET'S CONNECT</a>
```
Comment (Mihndim2020) [inline index.html]: - [ ] As per project requirements, this should be in a `<header>` tag, please kindly fix that.

File: index.html [hunk 7]
```diff
@@ -9,36 +9,214 @@
...
+        <div class="social-media">
+          <ul class="sm">
+            <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+            <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+            <li><a href="#" aria-label="gtihub"><i class="fab fa-github"></i></a></li> 
+          </ul>
+        </div>
+      
+        <button class="see">Get My Resume</button>
```
Comment (Mihndim2020) [inline index.html:178]: - [ ] The button should follow the button styling: Active, Hover and Pressed (see design below) and kindly fix that.
buttonsat

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @direakanbi, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  The linters are passing
:heavy_check_mark: The project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below and under the review._

- [ ] Please make sure your PR and commit messages are descriptive enough, checkout this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @direakanbi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**